packages/svelte/tests/runtime-runes/samples/class-private-raw-state/main.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+	class Counter {
+		#count = $state.raw(0);
```

Comment:
surely we need to use a mutable object here in order for this to actually test anything? ditto for the other tests. we should be testing a) that a mutation has no effect and b) that a replacement _does_ have an effect

Reply:
(e.g. [this](https://svelte-5-preview-git-state-raw-svelte.vercel.app/#H4sIAAAAAAAAE32QzW7DIBCEX2WFekjUiDRXalfqc8Q9OGQt0WJAsCSqEO9eMImS_qg3ZjXz7SyJTUpjYGKfmBlnZIK9Osc2jD5dFeGEmrDoYKOXddIF6ZWjl8EMpJHAHt5REvTwEGgk5H48rxJIGw0JeIK8fi7O4p2ikaSsgTlW32oNqY4HagC-JOCxh10NUP6R8uj0KH_Fyt4EnPOmNte935mwg3yFdttbf9MdIlGBWyO1kh99at3yclx7C0j3sIXQUn8TLj0b4iL-YZSfne1RTQqPTJCPmN_yF6Ai1DOUAQAA))

Reply:
> (e.g. [this](https://svelte-5-preview-git-state-raw-svelte.vercel.app/#H4sIAAAAAAAAE32QzW7DIBCEX2WFekjUiDRXalfqc8Q9OGQt0WJAsCSqEO9eMImS_qg3ZjXz7SyJTUpjYGKfmBlnZIK9Osc2jD5dFeGEmrDoYKOXddIF6ZWjl8EMpJHAHt5REvTwEGgk5H48rxJIGw0JeIK8fi7O4p2ikaSsgTlW32oNqY4HagC-JOCxh10NUP6R8uj0KH_Fyt4EnPOmNte935mwg3yFdttbf9MdIlGBWyO1kh99at3yclx7C0j3sIXQUn8TLj0b4iL-YZSfne1RTQqPTJCPmN_yF6Ai1DOUAQAA))

Hey Rich, sorry if I’m hijacking I just want clarity on this? So if I click mutate 3 times then click replace the number is 4. So is mutation being done internally but we only have a read only value?

Reply:
Rich hinted in the other comment that mutations to the object in DEV would result in warnings. You shouldn't do it as the change isn't tracked.